platform/features/conductor/src/ConductorRepresenter.js

```diff
@@ -159,8 +159,8 @@ define(
                 this.wireScope();
                 this.conductorElement =
                     this.$compile(TEMPLATE)(this.conductorScope());
-                this.element.after(this.conductorElement[0]);
-                this.element.addClass('l-controls-visible l-time-controller-visible');
+                this.element.parent().parent().after(this.conductorElement[0]);
+                this.element.parent().parent().addClass('l-controls-visible l-time-controller-visible');
```

Comment:
A little unfortunate this was necessary.

I think New Edit Mode could be handled with another Representer that does something similar to the time conductor, with ordering controlled by priority, such that you get a sensible order like:

* First, wrap the element in New Edit Mode markup
* Second, append a conductor

This would keep things nicely separate (so if, for example, we later decide we want the conductor _inside_ the glow, we just change priority order)

May need to add/amend the Representer API to support this. In #69, I'd really like to get some better API around views (extension categories `templates`, `representations`, and `views` collectively) such that we can interact with them from JavaScript more readily; I think this will simplify this kind of behavior.